services/bots/src/cla-sign/cla-sign.service.ts

```diff
@@ -58,7 +60,10 @@ export class ClaSignService {
     ).Item;
 
     if (!pendingRequest) {
-      throw new ServiceError('No pending request', { data: { signData }, service: 'cla-sign' });
+      throw new ServiceRequestError('No pending request', {
```

Comment:
Can we clarify this message, something like
```suggestion
      throw new ServiceRequestError(`No pending request found for ${signData.github_username}`, {
```

Reply:
Makes sense 👍